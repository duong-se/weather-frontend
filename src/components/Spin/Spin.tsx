import React from 'react'

export const Spin: React.FC = () => {
  return (
    <div className="spinner-border spinner-border-sm">
      <span className="sr-only" />
    </div>
  )
}

export default Spin
