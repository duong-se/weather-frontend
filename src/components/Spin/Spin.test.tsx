import React from 'react'
import { shallow } from 'enzyme'
import Spin from './Spin'

describe('Spin', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Spin />)
    expect(wrapper).toMatchSnapshot()
  })
})
