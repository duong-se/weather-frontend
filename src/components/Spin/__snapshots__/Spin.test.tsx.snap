// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spin should match snapshot 1`] = `
<div
  className="spinner-border spinner-border-sm"
>
  <span
    className="sr-only"
  />
</div>
`;
